src/scss/grommet-core/_objects.card.scss

```diff
@@ -50,14 +57,14 @@ $card-hover-color: #EBEBEB;
   //     }
   //   }
   //
-  //   .#{$grommet-namespace}control-icon-watch {
-  //     @include media-query(palm) {
-  //       // don't shrink video WatchIcon when in mobile
-  //       width: round($inuit-base-spacing-unit * 6);
-  //       height: round($inuit-base-spacing-unit * 6);
-  //     }
-  //   }
-  // }
+    .#{$grommet-namespace}control-icon-watch {
```

Comment:
@ericsoderberghp 
Added responsive option to icons in grommet-icon-loader: https://github.com/grommet/grommet-icon-loader/pull/4

Also removed WatchIcon mobile styles from Card styles, and updated Icon styles:
https://github.com/grommet/grommet/pull/852/commits/97f131d20cb717cd56f81d609dc6572c77136192#diff-d03f8eb3b97425583c39fc4f55957b12R72